service-api/module/Application/src/Fixtures/DataImportHandler.php

```diff
@@ -77,4 +78,46 @@ public function insertData(string $tablename, array $item): void
             ]);
         }
     }
+
+    public function updateCaseData(string $uuid, string $attrName, string $attrType, string $attrValue): void
+    {
+        $idKey = [
+            'key' => [
+                'id' => [
+                    'S' => $uuid,
+                ],
+            ],
+        ];
+
+        try {
+            $this->updateItemAttributeByKey('cases', $idKey, $attrName, $attrType, $attrValue);
+        } catch (AwsException $e) {
+            $this->logger->error('Unable to update data [' . $e->getMessage() . '] for case' . $uuid, [
+                'data' => [$attrName => $attrValue]
+            ]);
+        }
+    }
+
+    public function updateItemAttributeByKey(
```

Comment:
I don't think this function is used anywhere other than `updateCaseData`, could they just be merged together?

Reply:
Yeah makes sense to condense this, thought might have other variations of use but not really needed